pom.xml

```diff
@@ -38,7 +38,7 @@
 
         <github.url>scm:git:git@github.com:dockstore/dockstore-cli.git</github.url>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <dockstore-core.version>1.15.0-alpha.11</dockstore-core.version>
```

Comment:
Will make a proper release after merge of main PR (or possibly during staging release tagging)